Easydict/objc/Service/Language/EZLanguageModel.m

```diff
@@ -156,9 +157,18 @@ @implementation EZLanguageModel
         portuguese.localName = @"Português";
         portuguese.flagEmoji = @"🇵🇹";
         portuguese.localeIdentifier = @"pt_PT";
-        portuguese.voiceName = @"Luciana";
+        portuguese.voiceName = @"Joana";
```

Comment:
Here Portugal has the wrong voiceName, I changed it to `Joana`.

A lot of this code is sample code, I wrote it with the help of ChatGPT, so I can't guarantee that it's correct 😥
 
<img width="1673" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/bedbed0d-9c85-479e-858c-0f616e49f7fe">